### PR TITLE
styles(CheckoutPage): keep loading state while page transition doesn't get into place

### PR DESCRIPTION
**Pull request by guilhermespopolin**
## Describe your changes

* Keep loading state while page is transitioning

https://user-images.githubusercontent.com/19200662/228395689-4f881a6d-f743-4e76-bb30-f31a42a737f9.mov

## Jira issue(s): [GRW-2216](https://hedvig.atlassian.net/browse/GRW-2216)

[GRW-2216]: https://hedvig.atlassian.net/browse/GRW-2216?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by guilhermespopolin** [comment]
Current dependencies on/for this PR:
* main
  * **PR #2062** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/2062" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/2062?utm_source=stack-comment).

File: apps/store/src/components/CheckoutPage/useHandleSubmitCheckout.ts [hunk 1]
```diff
@@ -37,7 +37,7 @@ export const useHandleSubmitCheckout = (options: Options) => {
     userError = { message: t('UNKNOWN_ERROR_MESSAGE') }
   }
 
-  const signLoading = [BankIdState.Starting || BankIdState.Pending || BankIdState.Success].includes(
+  const signLoading = [BankIdState.Starting, BankIdState.Pending, BankIdState.Success].includes(
```
Comment (guilhermespopolin) [inline apps/store/src/components/CheckoutPage/useHandleSubmitCheckout.ts:40]: It tuned out we have everything in place in order to make this work. What was stopping us was a small bug 🤦🏻

Comment (robinandeer) [reply apps/store/src/components/CheckoutPage/useHandleSubmitCheckout.ts:40]: haha whuuut that is kinda funnt

**Review by robinandeer — APPROVED** [review]
Very nice, such a simple fix!

Added via Giphy

**Pull request merged**